desktop/assets/texts/en-US/dialogs.ftl

```diff
@@ -1,2 +1,9 @@
 dialog-ok = OK
-dialog-cancel = Cancel
\ No newline at end of file
+dialog-cancel = Cancel
+
+picker-title-open-file = Open a single file
+picker-filter-supported = All Supported Files
+picker-filter-swf = SWF (*.swf)
+picker-filter-spl = FutureSplash Animator (*.spl)
+picker-filter-ruf = Ruffle Bundle (*.ruf)
```

Comment:
I wonder - what's the chance that these indeed can be translated in any meaningful way to something else than what they currently are? And even if so, do we want that?

Reply:
FutureSplash Animator probably can't be translated but could perhaps be transliterated

Ruffle Bundle could be translated ("Ruffle" should not, but "Bundle" I think could be)